src/makeCalculator.js

```diff
@@ -4,7 +4,45 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(number) {
+      this.result += number;
+    },
+
+    subtract(number) {
+      this.result -= number;
+    },
+
+    multiply(number) {
+      this.result *= number;
+    },
+
+    divide(number) {
+      if (number === 0) {
+        throw new Error('Cannot divide by zero');
+      }
+
+      this.result /= number;
+    },
+
+    reset() {
```

Comment:
The 'reset' method correctly resets the 'result' property to 0 and returns the calculator object for chaining as per the requirements.